tox.ini

```diff
@@ -6,7 +6,7 @@ skip_missing_interpreters = true
 deps=
   pytest
 
-commands=pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs:tests} --ignore=tests/functional
+commands=pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs:test} --ignore=test/functional
```

Comment:
In Python, although allowed, it is preferable to use `tests/` and not `test/` because `test` is a built-in Python module, which this would override. I would advise normalizing in `tests` everywhere if possible

Reply:
Thank you! I did that intentionally, not knowing that `test` is a built-in module.  I saw both `test` and `tests` in different repos, so I was trying to normalize - but to the wrong target. I'll switch that up.

Reply:
Good catch, you are right that there are other projects that use `test`. Now, there is nothing bad about leaving this as-is, but it is usually preferred to use `tests`. Overriding the module will not cause `anchore-cli` to break!